docs/docs/development/setup.md

```diff
@@ -421,19 +251,13 @@ All subsequent steps must be performed from the VS Code terminal _inside_ the co
 </TabItem>
 </OsTabs>
 
-#### Initialize West
+### Initialize the Application
```

Comment:
Might want to reduce the header level of "Step into the repository" above too, to match.

Reply:
Also we could just make above step two tabs `docker` and `others`, to reduce duplication.

Reply:
Unfortunately, if we reduce the tabs to just "docker" and "others", then it can't sync your OS selection with the other tab groups.

Reply:
> Might want to reduce the header level of "Step into the repository" above too, to match.

Fixed.

Reply:
> Unfortunately, if we reduce the tabs to just "docker" and "others", then it can't sync your OS selection with the other tab groups.

That's true, although we already have a few other tabs that just divide by Linux/Windows/mac OS etc.